webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests
+
+def test_numbers_link(session):
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href=%s>123456</a>" % url)
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    assert session.url == "http://web-platform.test:8000%s" % url
+
+
+def test_multi_line_link(session):
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("""
+        <p style="background-color: yellow; width: 50px;">
+            <a href=%s>Helloooooooooooooooooooo Worlddddddddddddddd</a>
+        </p>""" % url)
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    assert session.url == "http://web-platform.test:8000%s" % url
+
+
+def test_link_unload_event(session):
```

Comment:
Mind moving all the link + navigate tests into a separate file like `navigate.py`?

Reply:
fixed